pkg/platform/model/vcs_test.go

```diff
@@ -152,7 +152,7 @@ func (suite *VCSTestSuite) TestVersionStringToConstraints() {
 	}
 
 	for _, tt := range tests {
-		got, err := VersionStringToConstraints(tt.version)
+		got, err := versionStringToConstraints(tt.version)
```

Comment:
I got confused between this PR and #2659. This fix was destined for that PR and should have been re-merged into this PR so that it would not show up. Oh well.